walletsetup.go

```diff
@@ -373,6 +373,16 @@ func promptConsoleSeed(reader *bufio.Reader) ([]byte, error) {
 		}
 
 		seedStrTrimmed := strings.TrimSpace(seedStr)
+		wordCount := strings.Count(seedStrTrimmed, " ") + 1
+
+		if wordCount == 1 {
+			seed, err := hex.DecodeString(seedStrTrimmed)
+			if err != nil {
+				fmt.Printf("Input error: %v\n", err.Error())
+				continue
+			}
+			return seed, nil
```

Comment:
I'd declare seed above the if statement, and then turn this into an if else.  If it's 1, try hex and assign seed if it is. otherwise, try the word list and assign seed if it decodes correctly.  If it's none of these, or the seed is too short, that will be detected below when it checks the seed length.